salt/scripts.py

```diff
@@ -156,7 +156,7 @@ def escalate_signal_to_process(pid, signum, sigframe):  # pylint: disable=unused
         signal.signal(signal.SIGTERM, prev_sigterm_handler)
 
         if not process.exitcode == salt.defaults.exitcodes.SALT_KEEPALIVE:
-            break
```

Comment:
This here is the **Big Change** that fixes *most* of problems of the exit status of ``salt-minion`` being forced to ``0``.